build-logic/convention/src/main/kotlin/ModuleConventionPlugin.kt

```diff
@@ -0,0 +1,23 @@
+import com.android.build.gradle.LibraryExtension
+import com.dhc.buildlogic.util.configureFlavor
+import com.dhc.buildlogic.util.configureKotlinAndroid
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.configure
+
+class ModuleConventionPlugin : Plugin<Project> {
+    override fun apply(target: Project) {
+        with(target) {
+            with(pluginManager) {
```

Comment:
ModuleConventionPlugin에는 보통 어떤걸 세팅해야하는거야?

Reply:
@ddyeon 
core랑 data쪽에서 동일한 gradle 세팅을 갖고 있길래 공통화 시켜봤는데, 
안드로이드 의존성이 필요한데, application이나 presentation이 아닌 모듈에 적용이 될듯함.

Reply:
```suggestion
// Android 의존성이 필요하지만 Application 이나 presentation 이 아닌 모듈에 사용
class ModuleConventionPlugin : Plugin<Project> {
    override fun apply(target: Project) {
        with(target) {
            with(pluginManager) {
```
오호 헷갈리지 않도록 주석을 적어보는 것두 좋을듯?!